testsettings.gradle

```diff
@@ -9,7 +9,7 @@ tasks.withType(Test) {
      * anything else             : run the non-cloud tests
      */
     String TEST_TYPE = "$System.env.TEST_TYPE"
-
+    forkEvery 50
```

Comment:
50 seems much better, though its a little hard to tell since there is so much variability in the normal case.